components/brave_wallet_ui/components/desktop/views/portfolio/components/token-lists/virtualized-tokens-list.tsx

```diff
@@ -5,7 +5,7 @@
 
 import * as React from 'react'
 import { VariableSizeList as List } from 'react-window'
-import AutoSizer from '@brave/react-virtualized-auto-sizer'
```

Comment:
don't need this file's change

Reply:
Master was breaking when I submitted this PR so I temporarily based the branch on the fix branch